CHANGELOG.md

```diff
@@ -1,5 +1,43 @@
 # Changelog
 
+
+## [v1.7.0-rc.0](https://github.com/kubeflow/training-operator/tree/v1.7.0-rc.0) (2023-07-07)
+
+**Breaking Changes**
+- Make scheduler-plugins the default gang scheduler. [\#1747](https://github.com/kubeflow/training-operator/pull/1747) ([Syulin7](https://github.com/Syulin7))
```

Comment:
I think this is not a breaking change since the operator will go in the following if passing a value excepted `volcano` and `scheduler-plugins` to `--gang-scheduler-name`:

Before this PR: The operator wouldn't take any action for the gang-scheduling.
After this PR: The operator will use the `scheduler-plugins`.

WDYT?